services/web-app/components/medium-posts/medium-posts.module.scss

```diff
@@ -0,0 +1,42 @@
+@use '@carbon/react/scss/spacing' as spacing;
+@use '@carbon/react/scss/type' as type;
+@use '@carbon/react/scss/breakpoint' as breakpoint;
+
+.medium-posts-container {
+  margin-top: spacing.$spacing-06;
+}
+
+// Fixes the breaking type
+.card-container h4 {
+  @include breakpoint.breakpoint('sm') {
+    @include type.type-style('heading-03');
+  }
+
+  @include breakpoint.breakpoint('md') {
+    font-size: 1rem;
+    font-weight: 400;
+    letter-spacing: 0;
+    line-height: 1.2rem;
+  }
+
+  @include breakpoint.breakpoint('lg') {
+    font-size: 0.75rem;
+    font-weight: 400;
+    letter-spacing: 0;
+    line-height: 1rem;
+  }
+  @include breakpoint.breakpoint('xlg') {
+    font-size: 1rem;
+    font-weight: 400;
+    letter-spacing: 0;
+    line-height: 1.375rem;
+  }
+}
```

Comment:
```suggestion
```

Do we need these lines? I removed them locally and didn't see any difference? What am I missing? I think we should be good to just pull styles directly from Article Card.